kubectx.plugin.zsh

```diff
@@ -13,6 +14,19 @@ if [[ ! -f "${KUBECTX_DIR}/completion/kubectx.zsh" ]]; then
   popd
 fi
 
+function loadCompletions()
+{
+  COMPLETION_DIR="${OH_MY_ZSH_DIR}/completions"
```

Comment:
Rather than creating a new completions directory, it would be better to add the completions to `$fpath` by doing something like:

```
fpath=($KUBECTX_DIR/completion $fpath)
```

instead of creating the `loadCompletions` function.

Reply:
I now saw the instruction about $fpath in the original instructions here: https://github.com/ahmetb/kubectx#linux
Please check the updated pull request

Reply:
Perfect, thanks!